docs/learn/universal-receiver/accept-reject-assets.md

```diff
@@ -8,35 +8,39 @@ import TabItem from '@theme/TabItem';
 
-![LSP1UniversalReceiverDelegate-Guide](/img/guides/lsp1/UniversalReceiverDelegate-Guide.jpeg)
+| 📢 Type ID to notify when receiving an LSP7 token 🪙                 |
+| :------------------------------------------------------------------- |
+| `TYPEID_LSP7_TOKENSRECIPIENT`                                        |
+| `0x429ac7a06903dbc9c13dfcb3c9d11df8194581fa047c96d7a4171fc7402958ea` |
 
-## Rejecting all Assets
+| 📢 Type ID to notify when receiving an LSP8 NFT 🖼️                   |
+| :------------------------------------------------------------------- |
+| `TYPEID_LSP8_TOKENSRECIPIENT`                                        |
+| `0x20804611b3e2ea21c480dc465142210acf4a2485947541770ec1fb87dee4a55c` |
 
-In order to **reject all the assets** that are being transferred to the profile, we need to create a Universal Receiver Delegate contract that reverts when it's the case of asset transfer (LSP7 & LSP8). The [`typeId`](../../contracts/contracts/LSP0ERC725Account/LSP0ERC725Account.md#universalreceiver) is the parameter that will give us more context on the call being made.
+:::success TypeIds available
 
-_e.g._
+A full list of LSP1 TypeIds that can be filtered from the `UniversalReceiver` event can be found under the [**contract > Universal Receiver TypeIds**](../../contracts/type-ids.md)
 
-- If `typeId` is **`0x429ac7a06903dbc9c13dfcb3c9d11df8194581fa047c96d7a4171fc7402958ea` \_TYPEID_LSP7_TOKENSRECIPIENT**, then we know that we are receiving a LSP7 Token.
-
-- If `typeId` is **`0x20804611b3e2ea21c480dc465142210acf4a2485947541770ec1fb87dee4a55c` \_TYPEID_LSP8_TOKENSRECIPIENT**, then we know that we are receiving a LSP8 Token.
+:::
 
-### Deploy contract through Remix
+### 1 - Deploy contract via Remix
 
-The first step is to navigate to **[Remix's website](https://remix.ethereum.org/)** and create a new solidity file under the **contracts** folder.
+1. First go to the **[Remix's website](https://remix.ethereum.org/)**. Create a new solidity file `UniversalReceiverDelegate.sol` under the **contracts** folder.
 
 ![Creating Universal Receiver Delegate in Remix](/img/guides/lsp1/remix-creating-file.jpeg)
 
-After creating the **UniversalReceiverDelegate.sol** file, copy the code snippet below inside the file created. This code snippet will be responsible for rejecting all LSP7 & LSP8 assets being transferred to your profile.
+2. Copy the code snippet below inside the file. It contains the logic for rejecting any LSP7 & LSP8 assets being transferred to the Universal Profile.
 
-```sol title="UniversalReceiverDelegate.sol - Solidity Code snippet of the URD that reject all assets"
+```solidity title="UniversalReceiverDelegate.sol - Solidity Code snippet of the URD that reject any assets"
```

Comment:
I ran the code, it's not compiling, maybe it's related to the change of the package?

Reply:
Strange I tried on Remix on my end. It compiled successfully